img/logo/menu.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="15" viewBox="0 0 24 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0 13.5H24" stroke="#00343D" stroke-width="3"/>
+<path d="M0 1.5H24" stroke="#00343D" stroke-width="3"/>
+<path d="M0 7.5H24" stroke="#00343D" stroke-width="3"/>
+</svg>
```

Comment:
These SVGs are not optimized, you can use these app
For SVG: https://imageoptim.com/mac
For PNG, JPEG and among others: https://tinypng.com/
@Jefer-Max